package.json

```diff
@@ -16,7 +16,7 @@
     "mainnet:codegen": "npx graph codegen ./generated/mento.mainnet.subgraph.yaml",
     "mainnet:build": "npx graph build ./generated/mento.mainnet.subgraph.yaml",
     "mainnet:build-all": "npm run clean && npm run mainnet:compile && npm run mainnet:codegen && npm run mainnet:build",
-    "mainnet:deploy": "npx graph deploy --studio mento ./generated/mento.mainnet.subgraph.yaml"
+    "mainnet:deploy": "npx graph deploy --studio mento-celo ./generated/mento.mainnet.subgraph.yaml"
```

Comment:
have we documented anywhere how to access the subgraph? i remember we (temporarily?) lost access to the first one, then I set up another one from the dev wallet, now this is the 3rd one I see

would be good to update the respective README section https://github.com/mento-protocol/mento-subgraph/?tab=readme-ov-file#how-do-i-access-the-subgraph-dashboard-on-thegraphcom

Reply:
> have we documented anywhere how to access the subgraph? i remember we (temporarily?) lost access to the first one, then I set up another one from the dev wallet, now this is the 3rd one I see
> 
> would be good to update the respective README section https://github.com/mento-protocol/mento-subgraph/?tab=readme-ov-file#how-do-i-access-the-subgraph-dashboard-on-thegraphcom

Nothing has changed regarding how we access the subgraph; we will still use the dev wallet. When we set up a safe that we can use for shared access, we can update to use that.